### ISSUE TITLE
For in loops do not highlight correctly

### ISSUE THREAD

**Issue by ccampbell** [colinta/Swift-for-f-ing-sublime#11]
For example:

```swift
for thing in things {

}
```

Shows up as 

![qkyv](https://cloud.githubusercontent.com/assets/259316/26507072/b5364fbe-421c-11e7-84fb-ee278720c4c7.png)

**Comment by colinta** [comment]
Fix it!  That's what PRs are for! 😃

**Comment by ccampbell** [comment]
Haha! I know sorry. It’s not a hard fix either. I’ll try to get around to it soon

**Comment by TheIronBorn** [comment]
I've fixed it locally by just adding `in` to one of the `keyword.control` sections.  Simple, though a bit dirty.

**Comment by TheIronBorn** [comment]
```regex
- match: \b(?>break|fallthrough|guard|try|catch|throws|return|in|is|as|case|continue|default)\b
```

**Comment by colinta** [comment]
Thanks!  Available in 2.3.9.  I was missing `throw`, with no "s", too.
